workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/routes/customer-portal/components/KeyTypeFilter/_key_type_filter.scss

```diff
@@ -0,0 +1,5 @@
+.filter-content {
```

Comment:
@pedroliveirav since there is only one child can you write the span on the same line, instead of nesting it?

Reply:
also, how come we need to use !important, in general it is best to avoid using it. How specific are the styles you are trying to override though?

Reply:
Done.

I need to change a Clay component style and, for me, it only worked using `!important`.
`.custom-control-outside .custom-control-label-text {`

Reply:
okay, we try to avoid using !important when we can, but for this seems alright, so i can send it through